src/ConnectionManager.js

```diff
@@ -69,6 +69,14 @@ define(["structures/Response", "structures/Request", "structures/CAPIError"],
             }
         }
 
+        // All other method types then defined in standard is sent as header to avoid issues
+        // with proxies, http security tools and limited web servers like the one in PHP.
+        var standardMethods = ["OPTIONS", "GET", "HEAD", "POST", "PUT", "DELETE", "TRACE"];
```

Comment:
nitpick: I would rather use an Object like in https://github.com/ezsystems/ez-js-rest-client/blob/master/src/authAgents/SessionAuthAgent.js#L88